src/praisonai-bot/praisonai_bot/gateway/server.py

```
@@ -4877,6 +4879,105 @@ def _refresh_metric_gauges(self) -> None:
+        """
+        gate = getattr(self, "_admission_gate", None)
+        watchdog = self._watchdog
+        outbox = self._scheduled_outbox
+        if gate is None and watchdog is None and outbox is None:
+            return None
+
+        admission_stats: Dict[str, Any] = {}
+        if gate is not None:
+            try:
+                admission_stats = gate.stats()
+            except Exception:  # pragma: no cover — defensive
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)